src/pages/access_review/VoteSummary.js

```diff
@@ -155,7 +155,8 @@ export const VoteSummary = hh(
           padding: '1rem',
           textAlign: 'right',
           flexShrink: '0',
-          fontWeight: Theme.font.weight.regular}
+          fontWeight: Theme.font.weight.regular},
+        isRendered: isNil(this.props.accessElection) ? false : this.props.accessElection.status === "Open"
```

Comment:
Have not tested different cases, but this might throw a null error on the election if the user visited this page for an unreviewed election or if the status is null ... not sure if that's a real case. Might be worth checking for validity first.

Reply:
there are three main cases - unreviewed where the election is null (the entire vote boxes are not rendered), or there is an election and it has a status that is either Open or Closed, but if there is an election the status is not null